lightning_uq_box/models/masked_ensemble/utils.py

```diff
@@ -23,44 +23,41 @@ def convert_deterministic_to_masked_ensemble(
             the interval [1, 6].
     """
     for name, value in list(deterministic_model._modules.items()):
-        if deterministic_model._modules[name]._modules:
+        assert value
+        if value._modules:
             convert_deterministic_to_masked_ensemble(
-                deterministic_model._modules[name],
-                num_estimators=num_estimators,
-                scale=scale,
+                value, num_estimators=num_estimators, scale=scale
             )
-        elif "Conv2d" in deterministic_model._modules[name].__class__.__name__:
-            curr_layer = deterministic_model._modules[name]
-            if curr_layer.in_channels >= 10:
+        elif isinstance(value, nn.Conv2d):
+            if value.in_channels >= 10:
                 setattr(
                     deterministic_model,
                     name,
                     MaskedConv2d(
                         num_estimators=num_estimators,
                         scale=scale,
-                        in_channels=curr_layer.in_channels,
-                        out_channels=curr_layer.out_channels,
-                        kernel_size=curr_layer.kernel_size,
-                        stride=curr_layer.stride,
-                        padding=curr_layer.padding,
-                        dilation=curr_layer.dilation,
-                        groups=curr_layer.groups,
-                        bias=curr_layer.bias is not None,
+                        in_channels=value.in_channels,
+                        out_channels=value.out_channels,
+                        kernel_size=value.kernel_size,  # type: ignore[arg-type]
+                        stride=value.stride,  # type: ignore[arg-type]
+                        padding=value.padding,  # type: ignore[arg-type]
+                        dilation=value.dilation,  # type: ignore[arg-type]
```

Comment:
Wasn't quite sure what was going on here, so didn't know how to fix it other than ignore it